app/javascript/components/student/editor/FileEditor.tsx

```diff
@@ -36,14 +79,43 @@ export const FileEditor = forwardRef<FileEditorHandle, FileEditorProps>(
     }))
 
     return (
-      <MonacoEditor
-        key={file.filename}
-        width="800"
-        height="600"
-        language="ruby"
-        editorDidMount={editorDidMount}
-        defaultValue={file.content}
-      />
+      <div>
+        <label htmlFor={`${file.name}-editor-wrap`}>Wrap</label>
+        <select
+          id={`${file.name}-editor-wrap`}
+          value={options.wordWrap}
+          onChange={handleWrapChange}
+        >
+          <option value="off">Off</option>
+          <option value="on">On</option>
+        </select>
+        <label>Theme</label>
+        <label htmlFor={`${file.name}-editor-theme`}>Theme</label>
+        <select
+          id={`${file.name}-editor-theme`}
+          value={theme}
+          onChange={handleThemeChange}
+        >
+          <option value="vs">Light</option>
+          <option value="vs-dark">Dark</option>
+        </select>
+        {content !== file.content && (
+          <button onClick={revertContent} type="button">
+            Revert to last run code
+          </button>
+        )}
+        <MonacoEditor
+          key={file.filename}
+          width="800"
+          height="600"
+          language={syntaxHighlighter}
```

Comment:
I think the name `syntaxHighlighter` isn't great for the _language_ prop.

Reply:
Agreed. Maybe something like `editorLanguage`?

Reply:
I'm going to use `editorLanguage`.